### PR TITLE
Add Wavelink to README

### PR DESCRIPTION
**Pull request by EvieePy**
Added version 3 of Wavelink to README.
Update previous Wavelink information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,23 +98,24 @@ Version numbers can come in different combinations, depending on the release typ
...
+|---------------------------------------------------------------------------|-----------------|--------------------------------------------|---------------------------------|
+| [Lavalink-Client](https://github.com/lavalink-devs/Lavalink-Client)       | Java/Kotlin/JVM | JDA/Discord4J/**Any**                      | Uses reactor                    |
+| [Lavalink.kt](https://github.com/DRSchlaubi/Lavalink.kt)                  | Kotlin          | Kord/JDA/**Any**                           | Kotlin Coroutines               |
+| [DisGoLink](https://github.com/disgoorg/disgolink)                        | Go              | **Any**                                    |                                 |
+| [Wavelink 3](https://github.com/PythonistaGuild/Wavelink/tree/feature/v3) | Python          | discord.py **V2**                          | `pip install -U wavelink --pre` |
+| [Mafic](https://github.com/ooliver1/mafic)                                | Python          | discord.py **V2**/nextcord/disnake/py-cord |                                 |
```
Comment (topi314) [inline README.md]:
```suggestion
| [Mafic](https://github.com/ooliver1/mafic)                                | Python          | discord.py **V2**/nextcord/disnake/py-cord |                                 |
| [Wavelink 3](https://github.com/PythonistaGuild/Wavelink/tree/feature/v3) | Python          | discord.py **V2**                          | `pip install -U wavelink --pre` |
```
Additional Information is not meant for install instructions, those should be contained in your readme

**Review by topi314 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EvieePy) [reply README.md]: I mean they are. But this makes it clear this is a pre-release install. But whatever.

Comment (topi314) [reply README.md]: just mention V3+, I see no issue

File: README.md [hunk 2]
```diff
@@ -98,23 +98,24 @@ Version numbers can come in different combinations, depending on the release typ
...
+|---------------------------------------------------------------------------|-----------------|--------------------------------------------|---------------------------------|
+| [Lavalink-Client](https://github.com/lavalink-devs/Lavalink-Client)       | Java/Kotlin/JVM | JDA/Discord4J/**Any**                      | Uses reactor                    |
+| [Lavalink.kt](https://github.com/DRSchlaubi/Lavalink.kt)                  | Kotlin          | Kord/JDA/**Any**                           | Kotlin Coroutines               |
+| [DisGoLink](https://github.com/disgoorg/disgolink)                        | Go              | **Any**                                    |                                 |
+| [Wavelink 3](https://github.com/PythonistaGuild/Wavelink/tree/feature/v3) | Python          | discord.py **V2**                          | Pre-Release (Version 3+)        |
+| [Mafic](https://github.com/ooliver1/mafic)                                | Python          | discord.py **V2**/nextcord/disnake/py-cord |                                 |
```
Comment (topi314) [inline README.md]:
```suggestion
| [Mafic](https://github.com/ooliver1/mafic)                                | Python          | discord.py **V2**/nextcord/disnake/py-cord |                                 |
| [Wavelink](https://github.com/PythonistaGuild/Wavelink/tree/feature/v3) | Python          | discord.py **V2**                          | Pre-Release (Version 3+)        |
```

**Pull request merged**